metrics-exporters/leadtime-exporter/lt-exporter.py

```diff
@@ -0,0 +1,141 @@
+import yaml
+import json
+import time
+import urllib3
+import requests
+import os
+from datetime import datetime, timezone
+from kubernetes import client, config
+from openshift.dynamic import DynamicClient
+from prometheus_client import start_http_server
+from prometheus_client.core import CounterMetricFamily,InfoMetricFamily,GaugeMetricFamily, REGISTRY
+
+class CommitCollector(object):
+    _prefix = "https://api.github.com/repos/"
+    _suffix = "/commits"
+    def __init__(self, username, token, projects, apps):
+        self._username = username
+        self._token = token
+        self._projects = projects
+        self._apps = apps
+    def collect(self):
+        ld_metric = GaugeMetricFamily('github_commit_timestamp', 'Commit timestamp', labels=['namespace', 'app', 'build', 'commit_hash'])
+        ld_metrics = generate_ld_metrics_list(projects)
+        for my_metric in ld_metrics:
+            print("Namespace: ", my_metric.namespace, ", App: ", my_metric.app_name, ", Build: ", my_metric.build_name)
```

Comment:
I would update this log to mention that we are performing a collection here.  This could be useful to track the activity of the collector.  It would also be useful to probably use something like this https://docs.python.org/3/library/logging.html so we get logging levels and timestamps.

Reply:
Using Logging library is a good idea, we can add this as an enhanecement, especially when we get have more exporters, Agree !